lib/Units.pm

```diff
@@ -180,6 +181,11 @@ our %known_units = (
 		factor => 1E-15,
 		m      => 1
 	},
+	meter => {
+		factor  => 1,
+		m       => 1,
+		aliases => ['meters']
+	},
```

Comment:
This should not be added as a new unit.  Instead "meter" and "meters" should be added directly as aliases of "m" (defined on line 42 of this file).